content/applications/websites/ecommerce/maximizing_revenue/reviews.rst

```diff
@@ -1,59 +1,70 @@
-===============================
-How to enable comments & rating
-===============================
+=========================
+Enable comments & ratings
+=========================
 
-Publishing and monitoring customer experience will help you gain the trust
-of new customers and better engage with your community. In 2 clicks, allow
-your customer to share their feedback!
+Properly publishing and accurately monitoring how your customers feel about the online shopping
+experience you provide is crucial for any business, especially if you want to organically gain
+the trust of your customer base and genuinely engage with your community.
+
+In just a couple of clicks, you can allow your customers to share their honest feedback, which you
+can use to make more informed business decisions going forward.
 
 .. image:: ./media/comment_post.png
    :align: center
+   :alt: sample comment on product page
 
 Setup
 =====
 
-Activate comments & rating from the *Customize* menu of the product web page.
+First, navigate to a product page, and select :menuselection:`Customize --> Discussion and
+Rating`. That will activate the comments and ratings feature.
 
-.. image:: ./media/comment_setup.png
+.. image:: ./media/comment-setup.png
    :align: center
+   :alt: customize discussion and rating setting
 
 .. note::
-    Visitors must log in to share their comments. Make sure they are able to
-    do so (see Portal documentation).
+    Visitors must log in to share their comments.
+    (see :doc:`../../../websites/ecommerce/shopper_experience/portal` ).
 
-Review the posts in real time
-=============================
+Review posts in real-time
+=========================
 
-Whenever a post is published, the product manager and all the product followers
-get notified in their Inbox (*Discuss* menu).
+Whenever a post is published, the product manager (and all the product followers)
+get notified about it. Also, the comment/review will show up in the *Chatter* on that product's
+detail page.
```

Comment:
For the use of the future tense inside of the documentation, please check the [documentation guidelines](https://www.odoo.com/documentation/user/14.0/contributing/documentation/content_guidelines.html#grammatical-tenses).